frame: encoder
sensor_topic: /encoder
odom_topic: /odometry/encoder
publish_rate: 100
pose_variance: [ 0.05, 0.05, 0.05 ]  # x, y, theta
velo_variance: [ 0.05, 0.05, 0.05 ]  # x, y, theta
